### DISCUSSION TITLE
Duda orden evento SEARCH-BY-AGE

### DISCUSSION THREAD

**Discussion by IgnacioHqz** [IIC2133-PUC/2024-2#139] [Tarea 2] [answered]
Hola! Puede que mi consulta sea algo obvia pero quisiera confirmar si estoy entendiendo bien.
El output de este evento son los id's de los pinguinos que tienen la edad AGE, pero la idea es que se ordenen de menor a mayor? Asumí eso, pero no sé si tal vez pueda ser el caso que se ordenen por orden de llegada. De todas formas perdón si es que llegase a ser obvia
Gracias!

**Comment by sebacarrascop** [comment, marked as answer]
hola!, se menciona que deben ordenarse por ID, no me quedo clara la pregunta.

**Reply by IgnacioHqz** [reply]
Hola! Estaba un poco confundido de si son los ids que se deben ordenar de menor a mayor o si luego de ordenar de menor a mayor la edad, mostrar los respectivos ids, pero por lo que estoy entendiendo debería ser lo segundo no? Gracias!
